LayoutTests/svg/hittest/svg-embedded-tooltip-expected.txt

```diff
@@ -1,7 +1,17 @@
 
 
 Verifying the inline SVG:
-PASS
+svg:
+	id:		main_svg
+	actual:		'outermost svg'
+	expected:	''
+
+use:
+	id:		use_without_title
+	actual:		'outermost svg'
+	expected:	''
+
+
```

Comment:
The test is here
https://searchfox.org/wubkat/source/LayoutTests/svg/hittest/svg-embedded-tooltip.html

The test is failing because even for inline SVG, the namespace URI is 

```
> # testing with inline SVG
> $0.ownerDocument instanceof Document
< true
> $0.tagName
< "svg"
> $0.namespaceURI
< "http://www.w3.org/2000/svg"
> $0.parentElement
< <body>…</body>
```

testing the `parentElement` would be better.